mcs/class/Makefile

```diff
@@ -105,6 +105,8 @@ monodroid_dirs := \
 	$(mobile_common_dirs)	\
 	Mono.CompilerServices.SymbolWriter	\
 	Mono.Btls.Interface	\
+	System.Runtime.Serialization.Formatters.Soap	\
+	System.Runtime.Remoting	\
```

Comment:
> XA ships whatever is in output folder

No longer true in the xamarin-android repo. We [explicitly list the assemblies we copy](https://github.com/xamarin/xamarin-android/blob/master/build-tools/mono-runtimes/ProfileAssemblies.projitems).

This doesn't answer whether or not we *should* include these assemblies in the Xamarin.Android profile. @kumpera tells me that "without [them] remoting is completely broken." I'm not sure how important that *actually* is, especially considering that iOS doesn't ship these assemblies either.

All I'm reasonably certain of is that we *do* support AppDomains, insofar  as we're able to allocate them and invoke code within the AppDomains, and I "know" that Remoting is *somehow* involved in AppDomains...so how "broken" are we, really, and does it matter?